templates/home.hbs

```diff
@@ -144,6 +144,9 @@
                         <h2>Connect</h2>
 
                         <ul class="connect-links">
+                            <li>
+                                <a href="{{site.info.base_url}}/cloud/changelog">Changelog <span class="external"></span></a>
+                            </li>
```

Comment:
Not sure if the changelog fits under the "connect" category.  Maybe in a future PR, we can move it to under the learn more maybe for cloud or somewhere in the footer.

Reply:
Agree - I will create an issue to revisit / remap the footer.